crates/rust-analyzer/src/config.rs

```diff
@@ -195,6 +224,9 @@ impl Config {
         set(value, "/completion/addCallParenthesis", &mut self.completion.add_call_parenthesis);
         set(value, "/completion/addCallArgumentSnippets", &mut self.completion.add_call_argument_snippets);
         set(value, "/callInfo/full", &mut self.call_info_full);
+        set(value, "/lens/run", &mut self.lens.run);
```

Comment:
let's also add `/lens/enable` which turns everything off (like we did for inlay hints)